.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,17 +43,10 @@ Steps to reproduce the behavior:
 4. Scroll down to '....'
 5. See error
 
-**Expected behavior**
```

Comment:
I think we should keep this. I added this originally because sometimes people were reporting things and I was unsure what they were expecting to happen. IMO this question helps people think a bit harder about what they are reporting and if its actually a bug.